google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -953,6 +953,13 @@ public static SignUrlOption withExtHeaders(Map<String, String> extHeaders) {
     public static SignUrlOption signWith(ServiceAccountSigner signer) {
       return new SignUrlOption(Option.SERVICE_ACCOUNT_CRED, signer);
     }
+    
+    /**
+    * Provides a host name to sign the URL. If not provided than host name will be default 
+    */ 
+    public static SignUrlOption withExtHostName(String extHostName){
```

Comment:
If we rename the option enum, we should also change this method name. Is the "ext" part important? Should it just be `withHostName`?